### ISSUE TITLE
`Mismatch in extents tracking stack` spam with a hanging `const` keyword

### ISSUE THREAD

**Issue by evilenzo** [godotengine/godot#76054]
### Godot version

4.0.2.stable

### System information

Windows 10

### Issue description

Engine keeps spamming me this
![изображение](https://user-images.githubusercontent.com/89089095/232018878-1584ed55-5867-4883-9519-67b82ef93813.png)

### Steps to reproduce

No steps, issue happens from time to time absolutely randomly

### Minimal reproduction project

N/A

**Comment by YuriSizov** [comment]
Without an example of a script that causes this we can't really investigate this.

**Comment by evilenzo** [comment]
> You can help the process by reducing your project until you can identify which part of it causes the error spam

Well, the problem is that this issue happened randomly couple times and my code changed since these moments so I don't really have an example that will immediatly trigger this issue. And that's ofc sad. But I can upload all the scripts that the project had at the moment the issue happened. Luckily there are only 3 small scripts since it's basic learn project. 
P.S. Problem happened in chunk.gd script that was attached to MeshInstance3D with ArrayMesh created in it. Other 2 scripts wasn't attached to anything.
Hope this helps!
[mrp.zip](https://github.com/godotengine/godot/files/11242713/mrp.zip)

**Comment by Diskane** [comment]
Hey I got the same bug. The issue seems to arise when writing the **const** keyword.

### **Godot version**
4.1.stable

### **System Information**
Windows 11

### **Issue description**
The parser is writing parser bug errors when the const keyword is written by itself no matter the scope
![image](https://github.com/godotengine/godot/assets/43890615/3ca96a34-7585-47ca-9e81-38323e7be36b)

### **Steps to reproduce**
Node3D + a new script attached to it

### **Minimal reproduction project**
N/A

I also tried it on my Android 11 smartphone with a Node2D, same problem.

**Comment by YuriSizov** [comment]
I can confirm, that hanging `const` on a line causes a bunch of following lines being printed:

```
modules/gdscript/gdscript_parser.cpp:4337 - Parser bug: Mismatch in extents tracking stack.
```

The line number is correct for 4.1-stable, it's in `GDScriptParser::complete_extents(Node *p_node)`.

**Comment by Drachenbauer** [comment]
For me it happens too, when i write the prefix "const" into an empty line.

**Comment by YuriSizov** [comment]
@Drachenbauer The issue has been fixed already and the fix will be available in future versions of the engine (4.1.2 and 4.2).
